test/testlog

```diff
@@ -3,2034 +3,2058 @@
 # Starting...
 # 339 test suites found.
```

Comment:
We usually don't push the testlog as it changes pretty much on every run, but I can take care of that change before I merge it, so no worries here.